pkg/api/getambassador.io/v3alpha1/roundtrip_test.go

```diff
@@ -0,0 +1,96 @@
+package v3alpha1
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// func TestAuthSvcRoundTrip(t *testing.T) {
```

Comment:
what's goin on here.....

Reply:
Right -- the `v3alpha1` roundtrip only tests the new things in `v3alpha1`. e.g. `AuthService` exists only in `v2`, so it's covered by the `v2` roundtrip test.

I've updated comments in #3681.